content/en/blog/2025/otel-arrow-phase-2.md

```diff
@@ -0,0 +1,66 @@
+---
+title: {{ replaceRE "[-_]" " " .Name | title }}
+linkTitle: OpenTelemetry Protocol with Apache Arrow Phase 2
+date: {{ dateFormat "2006-01-02" .Date }}
+author: >-
+  [Joshua MacDonald](https://github.com/jmacd) (Microsoft),
+  [Laurent Quérel](https://github.com/lquerel) (F5)
+draft: true
+issue: 6410
+sig: OTel-Arrow
+---
+
+## OTel-Arrow project announcement
+
```

Comment:
The frontmatter `title` appears at the top of the post, so no need to duplicate.

```suggestion

```